todo_project/utils/settings_config.py

```diff
@@ -0,0 +1,22 @@
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+DJANGO_ENV_VAR_NAME = "DJANGO_ENV"
+PRODUCTION = "PRODUCTION"
+DEVELOPMENT = "DEVELOPMENT"
+PRODUCTION_SETTINGS = "todo_project.settings.production"
+DEVELOPMENT_SETTINGS = "todo_project.settings.development"
+DEFAULT_SETTINGS = DEVELOPMENT_SETTINGS
+
+
+def configure_settings_module():
+    django_env = os.getenv(DJANGO_ENV_VAR_NAME, DEVELOPMENT).upper()
+
+    django_settings_module = DEVELOPMENT_SETTINGS
+
+    if django_env == PRODUCTION:
+        django_settings_module = PRODUCTION_SETTINGS
+
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", django_settings_module)
```

Comment:
Put this into config folder

Reply:
This is not the config for settings. 
This is a method to set the correct settings module in Django based on the environment

Reply:
This is setting config which will happen only once right?

Reply:
Right

Reply:
Anything which is for app config should not be in utils.

Reply:
Got it. Moved it to `settings` folder